src/keysignature.ts

```diff
@@ -158,18 +192,21 @@ export class KeySignature extends StaveModifier {
     }
   }
 
+  // Get the padding required for this modifier
   getPadding(index: number): number {
     if (!this.formatted) this.format();
 
-    return this.glyphs.length === 0 || (!this.paddingForced && index < 2) ? 0 : this.padding;
+    return this.children.length === 0 || (!this.paddingForced && index < 2) ? 0 : this.padding;
   }
 
+  // Get the width of the modifier
   getWidth(): number {
-    if (!this.formatted) this.calculateWidth();
+    if (!this.formatted) this.format();
```

Comment:
This one here is probably OK because it's trivial. No need for braces. But if we ever add an eslint rule to enforce braces, then I'm fine with braces showing up here too.

Reply:
I left it like it is as there are quite a few instances of
```
if (!this.formatted) this.format();
```
in the file. If you want, I can change all of them.